packagepoa/transform.py

```diff
@@ -222,10 +221,10 @@ def copy_pdf_to_output_dir(
             if file_content:
                 with open(os.path.join(output_dir, new_name), "wb") as out_handler:
                     out_handler.write(file_content)
-        except IOError:
+        except IOError as ioe:
             # The decap may return true but the file does not exist for some reason
             #  allow the transformation to continue in order to processes the supplementary files
-            LOGGER.error("decap returned true but the pdf file is missing %s", new_name)
+            LOGGER.error("decap returned true but the pdf file is missing %s: %s", new_name, ioe)
```

Comment:
this might give us some more insight in to the actual error.

Reply:
Agreed, I want to know more!